Autopilot/Inc/SensorFusion.hpp

```diff
@@ -30,6 +28,31 @@ struct SFOutput_t {
     float longitudeSpeed; //m/s
 };
 
+
+#ifndef ATTITUDE_DATATYPES_HPP //Temporary protection until redefinitions are deleted
+// New datatypes store the IMU and Airspeed data so SensorFusion and other modules
```

Comment:
Replace comment with "Following structs store the raw sensor data so other modules can have direct access to them without including sensor header files"